cypress-tests/README.md

```diff
@@ -108,59 +158,54 @@ In order to run cypress tests against multiple connectors at a time or in parall
-│   │   │   └── ...
-│   │   └── ConnectorUtils                               # Directory for utility functions related to connectors.
-│   │       ├── connector_detail_files_here.js
-│   │       └── utils.js
-│   ├── fixtures                                         # Directory for storing test data API request.
-│   │   └── your_fixture_files_here.json
-│   ├── support                                          # Directory for Cypress support files.
-│   │   ├── commands.js                                  # File containing custom Cypress commands and utilities.
-│   │   └── e2e.js
-│   └── utils
-│       └── utility_files_go_here.js
-├── cypress.config.js                                    # Cypress configuration file.
-├── cypress.env.json                                     # File is used to store environment-specific configuration values,such as base URLs, which can be accessed within your Cypress tests.
-├── package.json                                         # Node.js package file.
-├── readme.md                                            # This file
-└── yarn.lock
-```
+## Folder structure
 
-## Writing Tests
+The folder structure of this directory is as follows:
 
-### Adding Connectors
+```txt
+.
+├── .prettierrc                       # prettier configs
+├── README.md                         # this file
+├── cypress
+│   ├── e2e
+│   │   ├── <Service>Test             # Directory for test scenarios related to connectors.
+│   │   │   ├── 00000-test_<0>.cy.js
+│   │   │   ├── ...
+│   │   │   └── 0000n-test_<n>.cy.js
+│   │   └── <Service>Utils            # Directory for utility functions related to connectors.
+│   │        ├── connector_<1>.js
+│   │        ├── ...
+│   │        └── connector_<n>.js
+│   ├── fixtures                      # Directory for storing test data API request.
+│   │   ├── fixture_<1>.json
+│   │   ├── ...
+│   │   └── fixture_<n>.json
+│   ├── support                       # Directory for Cypress support files.
+│   │   ├── commands.js               # File containing custom Cypress commands and utilities.
+│   │   ├── e2e.js
+│   │   └── redirectionHandler.js
+│   └── utils
+│       ├── RequestBodyUtils.js
+│       ├── State.js
+│       └── featureFlags.js
+├── cypress.config.js                 # Cypress configuration file.
+├── eslint.config.js                  # linter configuration file.
+└── package.json                      # Node.js package file.
+```
 
-To add a new connector for testing with Hyperswitch, follow these steps:
+## Adding tests
 
-1. Include the connector details in the `creds.json` file:
+### Addition of test for a new connector
 
-   example:
-
-   ```json
-   {
-     "stripe": {
-       "connector_account_details": {
-         "auth_type": "HeaderKey",
-         "api_key": "SK_134"
-       }
-     }
-   }
-   ```
+1. Include the connector details in the `creds.json` file
 
 2. Add the new connector details to the ConnectorUtils folder (including CardNo and connector-specific information).
```

Comment:
In the following paragraph, can we link to the `Stripe.js` file instead of having the file path as a code block?